DonAlexandro/schedule-builder
# feature/signup-flow

## Overview
This PR will add singup flow with email confirmation

## Test cases
1. Go to signup page
2. Fill inputs with valid credentials
3. Press Sign Up button
4. After a couple of seconds, you must see a message that you need to confirm your email
5. Go to your inbox
6. Click on the link
7. Your email must be confirmed and you must be redirected to dashboard with notification telling that operation was successful.

## Trello ticket
https://trello.com/c/vZ5Z5kcw/1-basic-authentication-flow
